app/styles/st-base/_layout.scss

```diff
@@ -1,4 +1,4 @@
-@import '../utils/_helpers.scss';
```

Comment:
i think we will still need to import the ` _helpers.scss` file as the _layout.scss file still references some vars from _layout.scss

Reply:
That's odd. I see the dependent variables you mention, but prior to a change a week ago, this file wasn't actually included in this .scss file at all - even though it contained these variables.

I'll try to extract those variables from `_helpers.scss` as well.

Reply:
I've moved the section containing those dependent variables from `_helpers` to a different file. I could be even more minimal about this change and only move the variable declarations.

Reply:
i see. that's strange. i'm perplexed by the .scss tree of dependencies sometimes.